partitura/utils/fluidsynth.py

```diff
@@ -0,0 +1,326 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+"""
+This module contains methods for synthesizing score- or performance-like
+objects using fluidsynth. Fluidsynth is an optional dependency.
+"""
+
+import os
+from collections import defaultdict
+from typing import Callable, Optional, Union
+
+import numpy as np
+import partitura as pt
+
+try:
+    from fluidsynth import Synth
+
+    HAS_FLUIDSYNTH = True
+except ImportError:  # pragma: no cover
+    Synth = None  # pragma: no cover
+    HAS_FLUIDSYNTH = False  # pragma: no cover
+
+from partitura.utils.synth import SAMPLE_RATE
+from partitura.performance import PerformanceLike
+from partitura.score import ScoreLike
+from partitura.utils.misc import PathLike, download_file
+from partitura.utils.music import (
+    ensure_notearray,
+    get_time_units_from_note_array,
+    performance_notearray_from_score_notearray,
+)
+
+# MuseScore's soundfont distributed under the MIT License.
+# https://ftp.osuosl.org/pub/musescore/soundfont/MuseScore_General/MuseScore_General_License.md
+DEFAULT_SOUNDFONT_URL = "ftp://ftp.osuosl.org/pub/musescore/soundfont/MuseScore_General/MuseScore_General.sf3"
+
+DEFAULT_SOUNDFONT = os.path.join(
+    pt.__path__[0],
+    "assets",
+    "MuseScore_General.sf3",
+)
```

Comment:
I think it would be nice if such arguments were added to a globals.py script. This is started on PR #345 it can also be done later.

Reply:
This seems like a good idea

Reply:
I think it will better to do this on a later PR when all other PRs are merged.